src/helpers.ts

```diff
@@ -43,9 +43,9 @@ export function getTypeAndConversation(body: any): { type?: IncomingEventType; c
     const conversationId: string | undefined = (() => {
       let foundConversationId: string;
       if ('channel' in event) {
-        if (typeof event.channel === 'string') {
-          foundConversationId = event.channel;
-        } else if ('id' in event.channel) {
+        if (typeof event?.channel === 'string') {
+          foundConversationId = event?.channel;
```

Comment:
```suggestion
          foundConversationId = event.channel;
```

🔮 question: Is it safe to remove this since we know `foundConversationId` is a string?